.github/workflows/github_ci.yml

```diff
@@ -77,7 +77,7 @@ jobs:
     - uses: actions/checkout@v2
     - name: Build
       run: |
-        docker run --rm -v $(pwd):/src trzeci/emscripten make -f Makefile.simple emscripten
+        podman run --rm -v $(pwd):/src emscripten/emsdk make -f Makefile.simple emscripten
```

Comment:
Could you please update `.travis.yml` as well?

Reply:
Oh, sure, I've been meaning to give Travis some attention after solving the more outstanding issues with the GH CI. Shouldn't be hard to do using the latter as a template.